src/app/auth/auth.routes.ts

```diff
@@ -63,6 +64,7 @@ const routes: Routes = [
     SharedModule,
     AnnouncementModule,
     RecaptchaModule,
+    ComponentsModule,
```

Comment:
Since we're not using these components yet, I don't see a reason to include this module in any other modules. Hopefully they will only be imported when actually used to count down on unused imports.